rule-packs/gcp-privilege-escalation.json

```diff
@@ -190,5 +190,77 @@
       }
     ],
     "alertLevel": "MEDIUM"
+  },
+  {
+    "name": "gcp-possible-cloudbuilds-create-privilege-escalation-path",
+    "description": "Using the cloudbuilds.builds.create permission an attacker could potentially have access to additional read and write permissions to seven different GCP services (excluding Cloud Build itself). Most notably, they gain nearly-full access to Google Cloud Storage. It is recommended to restrict the permissions granted to the Cloud Build Service Account and to be careful granting the cloudbuild.builds.create permission to any users in your Organization.",
+    "queries": [
+      {
+        "name": "query0",
+        "query": "FIND google_iam_binding WITH permissions = 'coudbuilds.builds.create' AS permission THAT ASSIGNED >> (User | UserGroup) AS user RETURN user.displayName, user.email, permission.displayName ",
```

Comment:
Within the query (line 200), update 'coudbuilds.builds.create' to 'cloudbuilds.builds.create'

Reply:
fixed